extensions/stackdriver/stackdriver.cc

```diff
@@ -184,10 +184,17 @@ void StackdriverRootContext::onTick() {
   }
 }
 
-void StackdriverRootContext::record(const RequestInfo& request_info) {
+void StackdriverRootContext::record() {
   const auto peer_node_info_ptr = getPeerNode();
   const NodeInfo& peer_node_info =
       peer_node_info_ptr ? *peer_node_info_ptr : ::Wasm::Common::EmptyNodeInfo;
+  const auto& destination_node_info =
+      isOutbound() ? peer_node_info : local_node_info_;
+
+  ::Wasm::Common::RequestInfo request_info;
+  ::Wasm::Common::populateHTTPRequestInfo(isOutbound(), useHostHeaderFallback(),
+                                          &request_info,
+                                          destination_node_info.namespace_());
```

Comment:
I couldn't understand how could namepace_ function is defined. Could you share the knowledge?

Reply:
It is generated from proto: https://github.com/istio/proxy/blob/90b1d9f97e105d20698ece0ebd6fe2f75b44c7ea/extensions/common/node_info.proto#L27. Underscore is used to avoid reserved word `namespace`.

Reply:
`Underscore is used to avoid reserved word namespace.`
TIL